capsules/extra/src/eeprom.rs

```diff
@@ -0,0 +1,259 @@
+//! Driver for the AT24C32/64 EEPROM memory. Built on top of the I2C interface.
```

Comment:
Since we don't know how general this is, can we call this file `at24c32.rs`?

Reply:
right, however I'd stay without 32 at the end
when it comes to the capsule (struct) name I'm still thinking about `at24c_eeprom` or sth like that

Reply:
Either `at24c_eeprom` or something like `at24cxx_eeprom` seems totally reasonable. For the filename too.